provisioner/template.go

```diff
@@ -280,6 +281,8 @@ func renderTemplate(context *templateContext, filePath string) (string, error) {
 		"nodeCIDRMaxNodes": nodeCIDRMaxNodes,
 		"nodeCIDRMaxPods":  nodeCIDRMaxPods,
 		"parseInt64":       parseInt64,
+		"kmsKeyARN":        func(keyID string) (string, error) { return kmsKeyARN(context.awsAdapter, keyID) },
```

Comment:
For what do we need this function? Doesn't `kmsEncrypt` provide all we need or?

Reply:
You can't use an alias in an IAM policy, so we need a way to lookup a key using an alias. Technically `kmsEncrypt` returns both the ciphertext and the fully resolved key ARN, but I don't know if we should rely on it, plus it's going to look a bit magical in the templates.

Reply:
Ok makes sense. Didn't realize we needed this for the IAM role definition.